en/docs/manage/api-management.md

```diff
@@ -0,0 +1,54 @@
+# Manage
+
+Choreo's API Management capabilities allow you to manage your APIs through the Choreo Console. You can manage the lifecycle, consumers, and documents, secure your backends by applying rate--limiting policies through usage plans, define permissions using scopes, configure security, rate-limiting, and visibility settings, and define any properties. 
+
+# Overview
+
+The Overview of the Manage section gives you a bird's eye view of how you currently manage your API component. You can view the lifecycle status of your component, the number of consumers, the security scheme configured, and the usage plan applied to the API. The overview pane also lists the environment(s) of the API.
+
+# Lifecycle
+
+Choreo allows you to manage the lifecycle of the API through the API console. The states represent the stage where an API is in the process from creation to retirement. You can apply any of the six API lifecycles states: Created, Pre-released, published, blocked, deprecated, and retired while adhering to their respective lifecycle flows.
```

Comment:
Suggestion for the 2nd and 3rd sentences: "A state represents the stage of an API in the process from creation to retirement. You can apply any of the six API lifecycles states (i.e., created, pre-released, published, blocked, deprecated, and retired) while adhering to their respective lifecycle flows."

Suggesting this change because the capitalization method used for the states is currently not consistent, and because the 2nd sentence can be a bit ambiguous.

Reply:
Changed